incubator/nextcloud/values.yaml

```diff
@@ -0,0 +1,184 @@
+## Bitnami nextcloud image version
+## ref: https://hub.docker.com/r/bitnami/nextcloud/tags/
+##
+image:
+  registry: docker.io
+  repository: nextcloud
+  tag: latest
```

Comment:
Can you update the image information to follow our standard layout? See [the guide](https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#configuration) for more information and an example.

Also, please use a tagged version rather than latest. This will enable consistency between environments.